learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md

```diff
@@ -1,115 +1,209 @@
+
+| **Field**                   | **Value**                   | **Description**                                             |
+|-----------------------------|-----------------------------|-------------------------------------------------------------|
+| **Email**                   | 8404.john.smith@example.com | Use any test or fake emails                                 |
+| **Phone Number**            | +18004444444                | Use +18004444444 for phone number                           |
+| **OTP Verification Code**   | 000000                      | Use 000000 for the OTP verification code                    |
+| **First Name**              | John                        | Use any first name                                          |
+| **Last Name**               | Verified                    | Use Verified for the last name                              |
+| **Birthday**                | 01/01/1901                  | Use 01/01/1901 for successful identity verification         |
+| **Identification Type**     | Social Security Number      | Select Social Security Number for identification type       |
+| **Identification Number**   | 000000000                   | Enter 000000000 to fill the identification number field     |
+| **Country**                 | United States               | Select United States for country                            |
+| **Address Line 1**          | address_full_match          | Use address_full_match for successful identity verification |
+| **City**                    | Seattle                     | Use Seattle for city                                        |
+| **State**                   | Washington                  | Select Washington for state                                 |
+| **Zip Code**                | 12345                       | Use 12345 for zip code                                      |
+| **Test Credit Card Number** | 4242424242424242            | Use test credit card 4242424242424242                       |
+| **Expiration Date**         | 12/24                       | Use future expiration date 12/24                            |
+| **CVC**                     | 123                         | Use any CVC 123                                             |
+| **Billing Zip Code**        | 12345                       | Use any zip code 12345 for billing                          |
+
+### Onramp Kit KYC Test Data - Examples
+<img width="403" alt="Screen Shot 2023-03-29 at 4 32 26 AM" src="https://user-images.githubusercontent.com/9806858/228418052-30b2239a-ca19-4639-9858-4344d3ba7d45.png">
+<img width="392" alt="Screen Shot 2023-03-29 at 4 33 03 AM" src="https://user-images.githubusercontent.com/9806858/228418056-48cfa6a6-fde9-4504-a8be-ce91b03c960f.png">
+<img width="386" alt="Screen Shot 2023-03-29 at 4 34 25 AM" src="https://user-images.githubusercontent.com/9806858/228418059-b83b6357-a6b0-4f09-a4b2-3b89767cb4f0.png">
```

Comment:
`alt` should describe the image to help screen readers

Reply:
Thanks for the catch. Fixed: https://github.com/safe-global/safe-docs/pull/192/commits/9f225d29932aa7eb56fd4ed54fa74bf600f19228